websites/K/KissAsian/presence.ts

```diff
@@ -87,7 +87,7 @@ presence.on("UpdateData", async () => {
         }
       } else if (isNaN(duration)) {
         presenceData.startTimestamp = browsingStamp;
-        presenceData.details = "Looing at:";
+        presenceData.details = "Looikng at:";
```

Comment:
you made typo with another typo. It should be ``Looking``

Reply:
introducing a new typo!

Reply:
:|